Chapter3/db.erl

```diff
@@ -0,0 +1,38 @@
+-module(db).
+-export([new/0]).
+-export([destroy/1]).
+-export([write/3]).
+-export([delete/2]).
+-export([read/2]).
+-export([match/2]).
+
+new() -> [].
+destroy(Db) -> 0.
+write(Key, Element, Db) -> Db ++ [{Key, Element}].
```

Comment:
When your free can you explain this to me because I can't seem to get the lists to combine without using "++"